chat-plugins/polls.js

```
@@ -0,0 +1,102 @@
+/* Poll commands plugin
+ * This is a poll commands system that will allow a voice (or roomvoice) to host 
+ * polls in terms of starting, reminding
+ * by: panpawn
+ */
+
+function splint(target) {
+	var cmdArr = target.split(",");
+	for (var i = 0; i < cmdArr.length; i++) cmdArr[i] = cmdArr[i].trim();
+		return cmdArr;
+}
+
+exports.commands = {
+
+	survey: 'poll',
+	poll: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		if ((user.locked || user.mutedRooms[room.id]) && !user.can('bypassall')) return this.sendReply("You cannot do this while unable to talk.");
+		if (room.question) return this.sendReply('There is currently a poll going on already.');
+		if (!target) return false;
```

Silent failure? Bad. Call `/help poll` instead